features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/lifecycle.asset.js

```diff
@@ -298,7 +298,7 @@ $(function() {
 
 				var out = '<ul>';
 
-				var obj = JSON.parse(response);
+				var obj = response;
```

Comment:
removed redundant variable `obj` in commit 30deae1c9110153593e9e0db62fd72e81fccc44e of PR #240